firebase.js

```diff
@@ -91,6 +91,20 @@ class FirebaseController {
     return this.firebase.auth().signOut()
   }
 
+  async userNameExists() {
+    userId = this.firebase.auth().currentUser.uid
+    publicProfile = await this.firebase.database().ref('/users/' + userId + '/publicProfile').once('value')
```

Comment:
JS code styles recommend using `string${template}` for concatenation.
I broke this rule a couple of times for two-piece templates e.g.
```
`abc${def}`
'abc' + def
```
because it felt like + syntax was shorter. While writing this comment I realized it's the same number of characters, unless I exclude spaces :)
But in this PR you are using + here and a ${} template for the same ref later.